[2024-01-17 00:30:29,273] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_redshift.verificar_accesos manual__2024-01-17T00:30:21.601431+00:00 [queued]>
[2024-01-17 00:30:29,296] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_redshift.verificar_accesos manual__2024-01-17T00:30:21.601431+00:00 [queued]>
[2024-01-17 00:30:29,297] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-01-17 00:30:29,300] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2024-01-17 00:30:29,301] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-01-17 00:30:29,410] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): verificar_accesos> on 2024-01-17 00:30:21.601431+00:00
[2024-01-17 00:30:29,415] {standard_task_runner.py:52} INFO - Started process 5724 to run task
[2024-01-17 00:30:29,420] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_redshift', 'verificar_accesos', 'manual__2024-01-17T00:30:21.601431+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/ETLSachetto.py', '--cfg-path', '/tmp/tmpvov0en8d', '--error-file', '/tmp/tmp483i387_']
[2024-01-17 00:30:29,422] {standard_task_runner.py:80} INFO - Job 154: Subtask verificar_accesos
[2024-01-17 00:30:29,634] {task_command.py:371} INFO - Running <TaskInstance: dag_redshift.verificar_accesos manual__2024-01-17T00:30:21.601431+00:00 [running]> on host 8278623e2516
[2024-01-17 00:30:30,119] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=NachoSachetto
AIRFLOW_CTX_DAG_ID=dag_redshift
AIRFLOW_CTX_TASK_ID=verificar_accesos
AIRFLOW_CTX_EXECUTION_DATE=2024-01-17T00:30:21.601431+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-17T00:30:21.601431+00:00
[2024-01-17 00:30:33,675] {logging_mixin.py:115} INFO - Validaci√≥n Correcto
[2024-01-17 00:30:33,773] {python.py:173} INFO - Done. Returned value was: None
[2024-01-17 00:30:34,053] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_redshift, task_id=verificar_accesos, execution_date=20240117T003021, start_date=20240117T003029, end_date=20240117T003034
[2024-01-17 00:30:34,786] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-17 00:30:34,998] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
